docs/guides/expectations/advanced/how_to_add_comments_to_expectations_and_display_them_in_data_docs.md

```diff
@@ -1,5 +1,106 @@
 ---
 title: How to add comments to Expectations and display them in Data Docs
 ---
+import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
 
-This article is a stub.
+This guide will help you add descriptive comments (or notes, here used interchangeably) to Expectations and display those comments in Data Docs. In these comments you can add some clarification or motivation to the expectation definition to help you communicate more clearly with your team about specific expectations. Markdown is supported in these comments.
+
+<Prerequisites>
+
+- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
+- Configured a [Data Context](../../../tutorials/getting_started/initialize_a_data_context.md).
+- Configured an [Expectations Suite](../../../tutorials/getting_started/create_your_first_expectations.md).
+
+</Prerequisites>
+
+Steps
+-----
+
+1. **First, edit your Expectation Suite.**
+
+    ```bash
+    great_expectations --v3-api suite edit <your_suite_name>
+    ```
+
+2. **Next, add comments to specific Expectations.**
+
+    For each Expectation you wish to add notes to, add a dictionary to the ``meta`` field with the key ``notes`` and your comment as the value. Here is an example.
+
+    ```python
+    validator.expect_table_row_count_to_be_between(
+        max_value=1000000, min_value=1,
+        meta={"notes": "Example notes about this expectation."}
+    )
+    ```
+
+    Leads to the following representation in the Data Docs (click on the speech bubble to view the comment).
```

Comment:
Hey thanks for adding the change so quickly. I was thinking this is where we could have the note. 

```suggestion
    Leads to the following representation in the Data Docs (For Expectation Suite pages, click on the speech bubble to view the comment).
```

Reply:
What do you think? is it too subtle?